spec/models/spree/order_spec.rb

```
@@ -64,7 +78,7 @@
 
     context 'when user has newer orders' do
       before do
-        create(:order).update_columns(email: order.email, created_at: order.created_at + 1.minute)
+        create(:order).update(email: order.email, created_at: order.created_at + 1.minute)
```

Can you use `update!` here so that the setup code fails loudly in case there's a problem?